docs/build/smart-contracts/getting-started/storing-data.mdx

```diff
@@ -12,7 +12,7 @@ import TabItem from "@theme/TabItem";
 
 Now that we've built a basic Hello World example contract, we'll write a simple contract that stores and retrieves data. This will help you see the basics of Soroban's storage system.
 
-This is going to follow along with the [increment example](https://github.com/stellar/soroban-examples/tree/v22.0.1/increment), which has a single function that increments an internal counter and returns the value. If you want to see a working example, [try it in GitPod](https://gitpod.io/#https://github.com/stellar/soroban-examples/tree/v22.0.1).
+This is going to follow along with the [increment example](https://github.com/stellar/soroban-examples/tree/v22.0.1/increment), which has a single function that increments an internal counter and returns the value. If you want to see a working example.
```

Comment:
It's fine if you want to take out the gitpod link here, but as-is the sentence at the end just kinda hangs out there without an ending.